app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
I'd be fine with hardcoding `javascript,coffee,markdown,sql,yaml` for everyone if that's all we need right now.  Do we need one for `text` too?